gcc/testsuite/gcc.target/riscv/zcee-sext.c

```diff
@@ -1,5 +1,5 @@
 /* { dg-do compile { target { riscv64*-*-* } } } */
-/* { dg-options "-march=rv64gc_zcee -mabi=lp64 -O2" } */
+/* { dg-options "-march=rv64gc_zcee -mabi=lp64 -mzce-sext -O2" } */
```

Comment:
Same problem, If we need to alway use -mzce-sext to enable instructions or only need zcee in march